src/components/Header.astro

```diff
@@ -2,32 +2,73 @@
+        <path
+          d="M376.378 47.108V39.764H398.274V47.108H376.378Z"
+          fill="#15133C"
+        ></path>
+        <path
+          d="M451.2 36.704C451.2 41.736 450.18 46.2013 448.14 50.1C446.1 53.9533 443.266 56.968 439.64 59.144C436.013 61.32 431.82 62.408 427.06 62.408C422.345 62.408 418.174 61.32 414.548 59.144C410.921 56.968 408.088 53.9533 406.048 50.1C404.053 46.2467 403.056 41.804 403.056 36.772C403.056 31.6947 404.076 27.2293 406.116 23.376C408.156 19.4773 410.966 16.44 414.548 14.264C418.174 12.088 422.368 11 427.128 11C431.888 11 436.058 12.088 439.64 14.264C443.266 16.44 446.1 19.4547 448.14 23.308C450.18 27.1613 451.2 31.6267 451.2 36.704ZM442.088 36.704C442.088 33.168 441.476 30.108 440.252 27.524C439.028 24.94 437.305 22.9453 435.084 21.54C432.862 20.0893 430.21 19.364 427.128 19.364C424.09 19.364 421.438 20.0893 419.172 21.54C416.95 22.9453 415.228 24.94 414.004 27.524C412.78 30.108 412.168 33.168 412.168 36.704C412.168 40.24 412.78 43.3227 414.004 45.952C415.228 48.536 416.95 50.5307 419.172 51.936C421.438 53.3413 424.09 54.044 427.128 54.044C430.21 54.044 432.862 53.3413 435.084 51.936C437.305 50.4853 439.028 48.468 440.252 45.884C441.476 43.2547 442.088 40.1947 442.088 36.704Z"
+          fill="#15133C"
+        ></path>
+        <path
+          d="M453.793 25.756C453.793 22.8547 454.541 20.2933 456.037 18.072C457.533 15.8507 459.595 14.128 462.225 12.904C464.899 11.6347 467.982 11 471.473 11C474.827 11 477.729 11.5893 480.177 12.768C482.67 13.9467 484.597 15.624 485.957 17.8C487.362 19.976 488.11 22.56 488.201 25.552H479.497C479.406 23.4213 478.635 21.7667 477.185 20.588C475.734 19.364 473.785 18.752 471.337 18.752C468.662 18.752 466.509 19.364 464.877 20.588C463.29 21.7667 462.497 23.3987 462.497 25.484C462.497 27.252 462.973 28.6573 463.925 29.7C464.922 30.6973 466.463 31.4453 468.549 31.944L476.369 33.712C480.63 34.6187 483.803 36.1827 485.889 38.404C487.974 40.58 489.017 43.5267 489.017 47.244C489.017 50.2813 488.269 52.956 486.773 55.268C485.277 57.58 483.146 59.3707 480.381 60.64C477.661 61.864 474.465 62.476 470.793 62.476C467.302 62.476 464.242 61.8867 461.613 60.708C458.983 59.484 456.921 57.784 455.425 55.608C453.974 53.432 453.203 50.8707 453.113 47.924H461.817C461.862 50.0093 462.678 51.664 464.265 52.888C465.897 54.0667 468.095 54.656 470.861 54.656C473.762 54.656 476.051 54.0667 477.729 52.888C479.451 51.664 480.313 50.0547 480.313 48.06C480.313 46.3373 479.859 44.9773 478.953 43.98C478.046 42.9373 476.573 42.212 474.533 41.804L466.645 40.036C462.429 39.1293 459.233 37.4973 457.057 35.14C454.881 32.7373 453.793 29.6093 453.793 25.756Z"
+          fill="#15133C"
+        ></path>
+      </svg>
 
-    <span
-      class="
-        block
-        text-center
-      "
-    >
-      The Magento OS Community Alliance
-    </span>
-  </a>
+      <span
+        class="
+            block
+            text-center
+          "
+      >
+        The Magento OS Community Alliance
+      </span>
+    </a>;
```

Comment:
👀 I spy a little extra semi-colon

Reply:
True. That was part of the file you sent me and I thought it was something Astro-specific (never worked with Astro before) :joy:  If it is not, happy to remove it. Sec...

Reply:
Sorry 🙈